nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/test/java/org/nuxeo/ecm/automation/core/test/collections/GetCollectionsTest.java

```diff
@@ -77,26 +79,23 @@ public void testGetCollections() throws Exception {
 
         PaginableDocumentModelListImpl collectionsList = (PaginableDocumentModelListImpl) service.run(ctx, chain);
 
-        assertNotNull(collectionsList);
```

Comment:
Why? It doesn't hurt.

Reply:
because there is systematically an assert right after those checks that would fail if they were null. So why doing them ?

Reply:
To have an assertion message instead of a NPE.